alibi_detect/utils/tests/test_random.py

```diff
@@ -20,10 +23,39 @@ def test_fixed_seed(seed):
     """
     Tests the fixed_seed context manager.
     """
+    n = 5  # Length of random number sequences
+
+    nums0 = np.empty((n, 3))
```

Comment:
parametrize both `n` and `3` to make it clearer/more flexible?

Reply:
Ah I guess `3` is the number of distinct frameworks. But is a `numpy` container the most natural one for heterogenous contents...?

Reply:
Yeh its a fair point. I guess I could just unroll them all into a list? i.e. simply do `num0.append(...)` on all lines?